python/foglamp/services/core/api/common.py

```diff
@@ -86,7 +86,7 @@ def services_health_litmus_test():
     status_color = services_health_litmus_test()
     safe_mode = True if server.Server.running_in_safe_mode else False
 
-    return web.json_response({'uptime': since_started,
+    return web.json_response({'uptime': int(since_started),
```

Comment:
We have missed to fix tests for `$FogLAMP_ROOT/tests/unit/python/foglamp/services/core/api/test_common_ping.py`; as these were failing on local environment ONLY.
```
 assert 0.0 < content_dict["uptime"]
```
Let say Previously O/p of  `UPTIME.... 0.07508373260498047` And now `0`. So above assertion is failing as it executes fast ( and less than 1sec) on local setup.

Reply:
do you think better option will be `asyncio.sleep(1)` and verify `> 1` ?